src/session-managers/SecureSessionManager.ts

```diff
@@ -2,6 +2,11 @@
 import { FastifyRequest } from 'fastify'
 import { SerializeFunction } from '../Authenticator'
 
+// add this to prevent build errors
+// tests are passing, this is to backport this pr
+// https://github.com/fastify/fastify-passport/commit/43c82c321db58ea3e375dd475de60befbfcf2a11
```

Comment:
Please remove this message inside the code and put it inside the PR message.

Reply:
This comment was suggested by @mcollina. Its an important comment that explains why the eslint rule and the ts ignore

Reply:
Explaining why `eslint rule` is good. 

> this is to backport this pr https://github.com/fastify/fastify-passport/commit/43c82c321db58ea3e375dd475de60befbfcf2a11

But I don't think the last part is important for the rules. Instead, it should be part of PR message to show which commit is actually backported.

Reply:
Anyway, using [`@typescript-eslint/ban-ts-comment: allow-with-description`](https://typescript-eslint.io/rules/ban-ts-comment/#allow-with-description) is way better than disable the rules.